pkg/printers/internalversion/printers.go

```diff
@@ -1042,9 +1040,9 @@ func printStatefulSet(obj *apps.StatefulSet, options printers.PrintOptions) ([]m
 		Object: runtime.RawExtension{Object: obj},
 	}
 	desiredReplicas := obj.Spec.Replicas
-	currentReplicas := obj.Status.Replicas
+	readyReplicas := obj.Status.ReadyReplicas
```

Comment:
Since the idea is to merge `current/desired` to `ready`, I don't think this renaming is worthy.

Reply:
Sorry, but could you explain why to choose `obj.Status.ReadyReplicas` instead of `obj.Status.Replicas` here?

Reply:
I think it will make  ambiguous sometimes for users.
sometimes , the pod is not Ready, but the current and desired is equal. the Ready (1/1) will make users think the deployment is available. 
```
[root@hpa-vm:/etc/kubernetes]$ kubectl get deploy
NAME               READY   UP-TO-DATE   AVAILABLE   AGE
nginx-deployment   1/1     1           0           45m
```
OR we can using CURRENT instead of READY?
I dont think `Ready `is a good idea to show ` current/desired`

Reply:
Thanks, that's making sense now. I'll defer the choice to @kubernetes/sig-cli-feature-requests